x-pack/plugins/observability_solution/infra/server/lib/alerting/metric_threshold/metric_threshold_executor.ts

```diff
@@ -130,26 +134,29 @@ export const createMetricThresholdExecutor =
       throw new AlertsClientError();
     }
 
-    const alertReporter: MetricThresholdAlertReporter = async (
+    const alertReporter: MetricThresholdAlertReporter = async ({
       id,
       reason,
       actionGroup,
-      contextWithoutAlertDetailsUrl,
+      context: contextWithoutAlertDetailsUrl,
```

Comment:
what is this change about? 🤔

Reply:
I changed the `alertReporter` input type to be an object, to make it easier to pass fields and not think about the order, `context` is the field name in the params object.